pkg/kubelet/client/kubelet_client.go

```diff
@@ -72,7 +72,7 @@ func MakeTransport(config *KubeletClientConfig) (http.RoundTripper, error) {
 
 	rt := http.DefaultTransport
 	if config.Dial != nil || tlsConfig != nil {
-		rt = utilnet.SetTransportDefaults(&http.Transport{
+		rt = utilnet.SetOldTransportDefaults(&http.Transport{
```

Comment:
So, this is the client that master will use to contact kubelet directly? And this should only be for the SPDY uses we have?